base/shared/src/main/scala/scalaz/algebra/MonoidFunctions.scala

```diff
@@ -0,0 +1,7 @@
+package scalaz
+package algebra
+
+trait MonoidFunctions {
+  @inline final def empty[A](implicit A: Monoid[A]): A =
```

Comment:
See the build failure :)

Reply:
yep. I'll propose `mempty` and `mappend`.